container/20240111.md

```diff
@@ -0,0 +1,38 @@
+# 2024-01-11
+## :family: Participants (github handles preferred)
+
+* @paulphys
+* @mxmxchere
+* @jschoone
+* @batistein
+* @o-otte
+* @cah-hbaum
+* @artificial-intelligence
+* @matofeder
+* @berendt
+* @bitkeks
+* @michal-gubricky
+* @chess-knight
+* @DEiselt
+* @matfechner
```

Comment:
sorry, I forgot to add myself and Anurag also didn't add himself..
Can you please add
@janiskemper 
@kranurag7 
?
Thank you so much @matofeder and sorry again...

Reply:
no prob. d6325f32ea3c1ba57d3c2519c6df1cb8a45d2a30

Reply:
thanks!